service-jehlomat/src/main/kotlin/api/SyringeApi.kt

```diff
@@ -41,17 +41,29 @@ fun Route.syringeApi(database: DatabaseService, mailer: MailerService): Route {
         }
 
         post {
-            val dummyOrganization  = Organization(
-                1,
-                "TestOrg",
-                true
-            )
+            val syringe = call.receive<Syringe>()
+            val user = database.selectUserById(syringe.userId)
```

Comment:
Poslani trackovaciho kodu se tyka jen anonymnich uzivatelu:
https://cesko-digital.atlassian.net/browse/JH-36
v tom issue JH-41, to melo by lepe popsano. Tedy trackovaci emaily se posilaji jen neprihlasenym uzivatelu, ostatni si je prectou z profilu. Omlouvam se za komplikace.

Reply:
Rozdělil jsem funkčnost do dvou endpointů, jak jsme se bavili